_data-prepper/pipelines/configuration/processors/parse-json-processor.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: default
+title: Parse_JSON
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Parse JSON
+
+The `Parse_json` processor parses JSON data for an event, including any nested fields. The processor extracts the JSON pointer data and adds the input event to the extracted fields.
+
+
+## Configuration
+
+You can configure the `Parse_JSON` processor with the following options.
+
+| Option | Required | Type | Description |
+| :--- | :--- | :--- | :--- | 
+| `source` | No | String | The field in the `Event` that will be parsed. Default value is `message`. |
+| `destination` | No | String | The destination field of the parsed JSON. Defaults to the root of the `Event`. Cannot be `""`, `/`, or any whitespace-only `String` because these are not valid `Event` fields. |
+| `pointer` | No | String | A JSON pointer to the field to be parsed. There is no `pointer` by default, meaning the entire `source` is parsed. The `pointer` can access JSON Array indexes as well. If the JSON pointer is invalid then the entire `source` data is parsed into the outgoing `Event`. If the key that is pointed to already exists in the `Event` and the `destination` is the root, then the pointer uses the entire path of the key. |
```

Comment:
Are there specific values for this option? As in the example below: "outer_key/inner_key". Are there more? Or only this one?

Reply:
@dlvenable @chenqi0805 @cmanning09 @oeyh @asifsmohammed Are there specific values for this option?

Reply:
This is just an example format for JSON pointer. User can replace it with anything.